tests/js/test.js

```diff
@@ -200,6 +200,13 @@ describe('board.fen()', function () {
   });
 });
 
+describe('board.fen(showPromoted, countStarted)', function () {
+  it("it returns the current position in fen format, modified according to additional parameters", () => {
+    let board = new ffish.Board("makruk", "8/6ks/3M~2r1/2K1M3/8/3R4/8/8 w - 128 18 50");
+    chai.expect(board.fen(true, -1)).to.equal("8/6ks/3M~2r1/2K1M3/8/3R4/8/8 w - 128 0 50");
```

Comment:
If we give just one example, it might be confusing and people might think that setting `countStarted` to negative values is what users are supposed to do.
I suggest to add at least a single different example with a positive `countStarted` value.